src/lesson4/StringModification.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * @author ann
+ * @data 25.09.22
+ */
+package lesson4;
+
+public class StringModification {
+    public static String text = "ZюукяДZіботZ,ежобZощZяZенZ!ьлаксом";
+    public static String splitFrom = "Z";
+    public static String splitTo = " ";
+
+    public static void main(String[] args) {
+
+        String[] textAr = splitString(text, splitFrom);
+        String reversedText = reverseText(textAr, splitTo);
+        System.out.println(reversedText);
+    }
+    public static String[] splitString(String text, String splitFrom) {
```

Comment:
what is the point of this method, if you do split in main it will take one line of code